__tests__/app.test.js

```diff
@@ -322,3 +322,23 @@ describe('/api/articles', () => {
         })
       })
     })
+describe('/api/articles/:article_id (comment_count)', ()=> {
+  test('GET:200 sends the desired article with comment count added',()=> {
+    return request(app)
+    .get('/api/articles/3')
+    .expect(200)
+    .then((response) => {
+        expect(response.body.article).toMatchObject({
+            title: expect.any(String),
+            topic: expect.any(String),
+            body: expect.any(String),
+            author: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
Ideally we want to assert both the article_id (to ensure we have the right article back) and also the comment_count (to ensure it has been counted correctly)